vendor/bat-native-ads/src/bat/ads/internal/processors/contextual/text_embedding/text_embedding_html_events_database_table.h

```diff
@@ -0,0 +1,60 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_PROCESSORS_CONTEXTUAL_TEXT_EMBEDDING_TEXT_EMBEDDING_HTML_EVENTS_DATABASE_TABLE_H_
+#define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_PROCESSORS_CONTEXTUAL_TEXT_EMBEDDING_TEXT_EMBEDDING_HTML_EVENTS_DATABASE_TABLE_H_
+
+#include <functional>
+#include <string>
+
+#include "bat/ads/ads_client_callback.h"
+#include "bat/ads/internal/database/database_table_interface.h"
+#include "bat/ads/internal/processors/contextual/text_embedding/text_embedding_html_event_info.h"
+#include "bat/ads/public/interfaces/ads.mojom.h"
```

Comment:
`#include "bat/ads/public/interfaces/ads.mojom.h"` should be `#include "bat/ads/public/interfaces/ads.mojom-forward.h"` to use the forward declarations